src/LodeRunner.API/src/Core/Startup.cs

```diff
@@ -108,9 +109,10 @@ public static void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             if (env.IsDevelopment())
             {
+                int portNumber = AppConfigurationHelper.GetLoadRunnerUIPort(SystemConstants.LodeRunnerUIDefaultPort);
                 app.UseCors(builder =>
                 {
-                    builder.WithOrigins("http://localhost:3000")
+                    builder.WithOrigins($"http://localhost:{portNumber}")
```

Comment:
Will this work in a production deployment?  The UI will be in a different container making requests.  So, I think we'll have to somehow be able to configure for both local run and running when deployed in k8s.